lib/omniauth/strategies/latvija.rb

```diff
@@ -1,76 +1,76 @@
-require "time"
-require "rexml/document"
-require "rexml/xpath"
-require "openssl"
-require "xmlcanonicalizer"
-require "digest/sha1"
+require 'time'
+require 'rexml/document'
+require 'rexml/xpath'
```

Comment:
require `nokogiri` instead of `rexml`?

Reply:
maybe you can put the `xmlenc` require also here, so that all requires are in one place